src/Uno.UI/UI/Xaml/Application.macOS.cs

```diff
@@ -87,7 +87,17 @@ public override void DidFinishLaunching(NSNotification notification)
 			}
 			if (!handled)
 			{
-				OnLaunched(new LaunchActivatedEventArgs());
+				var argumentsString = "";
+				if (NSProcessInfo.ProcessInfo?.Arguments is { } processArguments)
+				{
+					// First argument is just executable path
+					if (processArguments.Length > 1)
+					{
+						argumentsString = string.Join(" ", processArguments.Skip(1));
```

Comment:
Can any element of `processArguments` contain a space? If yes, should it somehow be escaped, or the argument be enclosed in double quotes, etc?

Reply:
This should now not be a problem as the code is using `Environment.CommandLine` instead, which will give the verbatim string including any quotes, whitespace, etc., which matches UWP behavior